app/code/community/Bolt/Boltpay/controllers/ApiController.php

```diff
@@ -262,7 +262,7 @@ public function create_orderAction() {
                 array(
                     'status' => 'success',
                     'display_id' => $order->getIncrementId(),
-                    'total' => (int)($order->getGrandTotal() * 100),
+                    'total' => (int)round($order->getGrandTotal() * 100),
```

Comment:
Is this necessary? Does `$order->getGrandTotal()` return a value beyond two significant decimal digits?  Can we get an output showing a number here like "4.5943"?   I think that it will always be "4.59".